MapboxNavigation/NavigationMapView.swift

```diff
@@ -1006,7 +1006,8 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             }
             
             // Road labels should match road signage.
-            let locale = layer.sourceLayerIdentifier == "road_label" ? Locale(identifier: "mul") : nil
+            let isLabelLayer = MGLVectorTileSource.roadLabelLayerIdentifiersByTileSetIdentifier.values.contains(layer.sourceLayerIdentifier ?? "")
+            let locale = isLabelLayer ? Locale(identifier: "mul") : nil
```

Comment:
What's "mul"? Doesn't seem to be supported by `Locale` 🤔. If so, won't this always evaluate to `nil`?

Reply:
`mul` is the ISO&nbsp;639 identifier meaning “multilingual”. The map SDK maps this code to the `name` field. Since that field displays the local language for each feature, it effectively produces a multilingual map. `Locale(identifier: "mul")` is supported in the sense that its `languageCode` is `mul`, which is all we need for this API.